src/main/java/com/denizenscript/depenizen/bukkit/bridges/MythicMobsBridge.java

```diff
@@ -206,7 +340,40 @@ public static void setMythicVariableMap(Entity entity, Map<String, Variable> map
         registry.putAll(map);
     }
 
+    public static LocationTag locationFromAbstractLocation(AbstractLocation location) {
+        return new LocationTag(location.getWorld().getName(), location.getX(), location.getY(), location.getZ(), location.getYaw(), location.getPitch());
+    }
+
+    public static ItemExecutor getItemManager() {
+        return MythicBukkit.inst().getItemManager();
+    }
+
+    public static Collection<String> getItemNames() {
+        return getItemManager().getItemNames();
+    }
+
+    public static MythicItem getMythicItem(String name) {
+        return MythicBukkit.inst().getItemManager().getItem(name).orElse(null);
+    }
+
+    public static Collection<String> getSkillNames() {
+        return MythicBukkit.inst().getSkillManager().getSkillNames();
+    }
+
+    public static void setFaction(ActiveMob mob, String faction) {
+        mob.setFaction(faction);
+    }
+
+    public static String getFaction(ActiveMob mob) {
+        return mob.getFaction();
+    }
+
+    public static String parseMythic(AbstractEntity entity, String string) {
+        PlaceholderString placeholderString = PlaceholderString.of(string);
+        return placeholderString.get(entity);
```

Comment:
Same here, method is only used in one place and doesn't make the code significantly shorter / cleaner, can probably use `PlaceholderString.of(string).get(entity)` directly?